### PR TITLE
Document that removed state scripts recipes need to be cleaned up.

### PR DESCRIPTION
**Pull request by kacf**
Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

**Comment by kacf** [comment]
Related to result of https://github.com/mendersoftware/meta-mender/pull/459.

File: 04.Artifacts/07.State-scripts/docs.md [hunk 1]
```diff
@@ -100,6 +100,8 @@ The easiest way to have Mender run the state scripts is to create a new OpenEmbe
 
 Take a look at the [example-state-scripts](https://github.com/mendersoftware/meta-mender/tree/master/meta-mender-demo/recipes-mender/example-state-scripts) recipe to get started.
 
+!! If you add or remove a recipe containing state scripts to a build, you need to clear the `tmp` directory of the Yocto build before building a new image. An alternative is to call `bitbake -c clean <recipe>` with the affected recipe, but the first method is recommended since it will cover all cases, regardless of recipe name. Merely changing a recipe does not need this step.
```
Comment (estenberg) [inline 04.Artifacts/07.State-scripts/docs.md]: I think last sentence can be worded a bit differently, e.g. "Merely changing a recipe does not require this step."

Comment (kacf) [reply 04.Artifacts/07.State-scripts/docs.md]: Thanks, fixed!

**Comment by kacf** [comment]
Cherry-picked to 1.2.x and 1.3.x as well.

**Pull request merged**